src/utils/messageUtil.ts

```diff
@@ -12,8 +12,11 @@ export type Token = string | number | boolean;
 
 // TODO: Update util when sfdxdocgen is updated to support oclif
 export class MessageUtil {
+  private static messages = Messages.loadMessages('@salesforce/plugin-templates', 'messages');
```

Comment:
Changes here on down were all lint fixes - I updated types where I could, and removed linting where I couldn't. Other errors were for improper ordering, missing return types (those ones caused actual errors when fixed), and mostly missing types resulting in rogue `any`s.